From 6eedc5b64e7d94d4488923045173ce05c7d8881d Mon Sep 17 00:00:00 2001
From: Amit Kumar bhaumik <amitx.bhaumik@intel.com>
Date: Fri, 9 Sep 2022 08:05:50 +0000
Subject: [PATCH] Modified image layout for CTL

Updated image sections and offset as per ctl layout

Test:
Ctl image booting fine and new image layout reflected.

Signed-off-by: Amit Kumar bhaumik <amitx.bhaumik@intel.com>
Upstream-Status: Pending
---
 .../dts/openbmc-flash-layout-intel-128MB.dtsi | 34 +++++++------------
 1 file changed, 12 insertions(+), 22 deletions(-)

diff --git a/arch/arm/boot/dts/openbmc-flash-layout-intel-128MB.dtsi b/arch/arm/boot/dts/openbmc-flash-layout-intel-128MB.dtsi
index ca148cf9a364..491109f22f56 100644
--- a/arch/arm/boot/dts/openbmc-flash-layout-intel-128MB.dtsi
+++ b/arch/arm/boot/dts/openbmc-flash-layout-intel-128MB.dtsi
@@ -37,33 +37,23 @@ partitions {
 		label = "image-a";
 	};
 
-	rc-image@2a00000 {
-		reg = <0x2a00000 0x2000000>;
-		label = "rc-image";
-	};
-
-	image-staging@4a00000 {
-		reg = <0x4a00000 0x3400000>;
-		label = "image-stg";
-	};
-
-	afm-active@7e00000 {
-		reg = <0x7e00000 0x20000>;
-		label = "afm-active";
+	reserved@2a00000 {
+		reg = <0x2a00000 0x1600000>;
+		label = "rsvrd";
 	};
 
-	afm-recovery@7e20000 {
-		reg = <0x7e20000 0x20000>;
-		label = "afm-rcvr";
+	u-boot-b@4000000 {
+		reg = <0x4000000 0x80000>;
+		label = "u-boot-b";
 	};
 
-	reserved@7e40000 {
-		reg = <0x7e40000 0xc0000>;
-		label = "rsvrd";
+	pfm-b@4080000 {
+		reg = <0x4080000 0x20000>;
+		label = "pfm-b";
 	};
 
-	cpld-gold@7f00000 {
-		reg = <0x7f00000 0x100000>;
-		label = "cpld-gold";
+	fit-image-b@40a0000 {
+		reg = <0x40a0000 0x1f00000>;
+		label = "image-b";
 	};
 };
-- 
2.17.1

